sql/src/main/java/io/crate/planner/operators/Insert.java

```diff
@@ -95,7 +95,7 @@ ColumnIndexWriterProjection columnIndexWriterProjection() {
 
     @Override
     public List<Symbol> outputs() {
-        return MergeCountProjection.OUTPUTS;
+        return (List<Symbol>) writeToTable.outputs();
```

Comment:
Minor: The type of outputs is defined in multiple ways over the codebase, sometimes it is `List<Symbol>` sometimes it is `List<? extends Symbol>` and sometimes it is `List<Field>`. Are there any cases where the output is not a `Field `?

Reply:
Yes, there can be function symbols or Reference symbols as well.